SQL Scripts/CreateDB.sql

```diff
@@ -54,7 +54,10 @@ CREATE TABLE Vote(
 CREATE TABLE Contacts(
     owner_id Integer REFERENCES  Users(uid),
     contact_id Integer REFERENCES Users(uid),
+    first_name VARCHAR(30) NOT NULL,
```

Comment:
Is it not redundant to have this field here and in Users table?

Reply:
Nope, tu puedes ponerle el first name que tu quieras.